ResearchKit/Common/ORKResult.m

```diff
@@ -1836,6 +1837,60 @@ - (ORKStepResult *)stepResultForStepIdentifier:(NSString *)stepIdentifier {
 @end
 
 
+@implementation ORKLocationQuestionResult
+
+- (void)encodeWithCoder:(NSCoder *)aCoder {
+    [super encodeWithCoder:aCoder];
+    ORK_ENCODE_OBJ(aCoder, locationAnswer);
+}
+
+- (instancetype)initWithCoder:(NSCoder *)aDecoder {
+    self = [super initWithCoder:aDecoder];
+    if (self) {
+        ORK_DECODE_OBJ_CLASS(aDecoder, locationAnswer, MKPlacemark);
+    }
+    return self;
+}
+
++ (BOOL)supportsSecureCoding {
+    return YES;
+}
+
+- (BOOL)isEqual:(id)object {
+    BOOL isParentSame = [super isEqual:object];
+    
+    __typeof(self) castObject = object;
+    return (isParentSame &&
+            self.locationAnswer.location.coordinate.latitude == castObject.locationAnswer.location.coordinate.latitude &&
```

Comment:
It's not. Which I thought was strange. Calling isEqual: on MKPlacemark returns nil instead of true or false. I had a lot of trouble with this when I was writing the unit tests.